livekit-android-sdk/src/main/java/io/livekit/android/compose/VideoRenderer.kt

```diff
@@ -83,7 +83,12 @@ fun VideoRenderer(
 
     DisposableEffect(currentCompositeKeyHash.toString()) {
         onDispose {
-            view?.release()
+            view?.let {
+                it.release()
+                videoTrack?.removeRenderer(it)
```

Comment:
Hmm, is the `cleanupVideoTrack()` not getting called for you? That's essentially what this is doing, so it seems redundant.

Reply:
I'll test more. But only after these changes I stopped getting leak notification